lib/rest-client-view.coffee

```diff
@@ -91,6 +92,8 @@ class RestClientView extends ScrollView
           @textarea class: "field #{rest_form.headers.split('.')[1]}", rows: 7
           @strong 'User-Agent'
           @input class: "field #{rest_form.user_agent.split('.')[1]}", value: 'atom-rest-client'
+          @string 'Strict SSL'
+          @inplut type: 'checkbox', class: "field #{rest_form.strict_ssl.split('.')[1]}", checked: true
```

Comment:
also i've never seen a pound-sign in the class attribute, as pound-sign is always reserved for ID.  @javaguirre